Presscotts_Mortar_Calculator_(no_envirement)_V0.2.py

```diff
@@ -19,215 +18,225 @@
    air density 1pct incline]
 '''
 
-import math 
+import math
+
+
 def getList(dict):
     list = []
     for key in dict.keys():
         list.append(key)
-         
+
     return list
+
+
 def checkKey(dic, key):

```

Comment:
Unused function - can be trimmed for brevity.
This also just boils down to `if key in dic`.